packages/spectral/src/serverTypes/convertIntegration.ts

```diff
@@ -516,13 +516,21 @@ const convertConfigVar = (
           return result;
         }
 
+        const meta = convertInputPermissionAndVisibility(
+          pick(input, ["permissionAndVisibilityType", "visibleToOrgDeployer"]) as {
+            permissionAndVisibilityType?: PermissionAndVisibilityType;
+            visibleToOrgDeployer?: boolean;
+          },
+        );
+
```

Comment:
This needs to also be added to the `isConnectionReferenceConfigVar` clause below since we want to support setting permission and visibility for both connections defined in CNI and references of existing component connections.

Reply:
This has already been added to `isConnectionReferenceConfigVar` that uses `convertComponentReference` to compute the connection reference inputs.